shared/actions/unlock-folders.js

```diff
@@ -5,12 +5,7 @@ import engine from '../engine'
 import type {ToPaperKeyInput, OnBackFromPaperKey, CheckPaperKey, Finish,
   Waiting, RegisterRekeyListenerAction, NewRekeyPopupAction} from '../constants/unlock-folders'
 import type {TypedAsyncAction, AsyncAction, Dispatch} from '../constants/types/flux'
-import {createServer} from '../engine/server'
-import {delegateUiCtlRegisterRekeyUIRpc, loginPaperKeySubmitRpc, rekeyRekeyStatusFinishRpc,
-  rekeyShowPendingRekeyStatusRpc} from '../constants/types/flow-types'
-
-// We need the sessionID of the delegateRekeyUI sessions
-let rekeyServer = null
```

Comment:
after rekey refactor on daemon side this is supposedly no longer true. need to test this

- [ ] Test rekey flow